documentation/pe_puppetdb_notes.markdown

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
Is this release notes? Or rather, what is your overall goal for this file?

We should include an index change for this in _puppetdb_nav.html somewhere, so people can find it directly right?